packages/core/src/render3/i18n.ts

```diff
@@ -25,11 +25,11 @@ import {NO_CHANGE} from './tokens';
 import {addAllToArray, getNativeByIndex, getNativeByTNode, getTNode, isLContainer, stringify} from './util';
 
 const MARKER = `�`;
-const ICU_BLOCK_REGEX = /^\s*(�\d+�)\s*,\s*(select|plural)\s*,/;
+const ICU_BLOCK_REGEX = /^\s*(�\d+:?\d*�)\s*,\s*(select|plural)\s*,/;
```

Comment:
This will accept `�1:�`, is that a problem? Perhaps

```suggestion
const ICU_BLOCK_REGEX = /^\s*(�\d+(?::\d+)?�)\s*,\s*(select|plural)\s*,/;
```

which is more strict?

Reply:
@JoostK thanks for the comment. Strings that are parsed using these RegExps are generated by the compiler, so we should not have invalid patterns that you described (if we do - it should be checked/fixed on compiler side). Thank you.